src/Plivo/Version.cs

```diff
@@ -10,7 +10,7 @@ public class Version
         /// <summary>
         /// DotNet SDK version
         /// </summary>
-        public const string SdkVersion = "5.7.0";
+        public const string SdkVersion = "5.7.1";
```

Comment:
need to add version changes into version.json file as well 
reference: [version.json](https://github.com/plivo/plivo-dotnet/commit/0d4b75cd009efd659e22b72b272aeb2b15d5e51e)

Reply:
Done, please check.